spec/features/visitor_can_view_items_spec.rb

```diff
@@ -0,0 +1,35 @@
+# Background: I have several items and each of them has a title, description, price, and image
+#       I can see all existing items
+
+require "rails_helper"
+
+feature "View items" do
+
+    let!(:item_one) {Item.create(name: "Canon 7d",
+                                 description: "The best camera ever (NOT)",
+                                 price: 15)}
+
+    let!(:item_two) {Item.create(name: "Sony a7r",
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.